implementations/rust/message/src/message.rs

```diff
@@ -45,39 +44,15 @@ pub mod message {
     }
 
     impl Codec for Message {
-        type Inner = Message;
-        fn encode(msg: &Message, u: &mut Vec<u8>) -> Result<(), String> {
-            Route::encode(&msg.onward_route, u);
-            Route::encode(&msg.return_route, u);
+        type Inner = Box<Message>;
```

Comment:
Why a Box?

Reply:
Because I thought it was necessary due to how I was passing messages between modules and possibly threads. I've already removed the Box aspect (it's in the next PR).